bin/pycbc_optimal_snr

```diff
@@ -94,6 +94,13 @@ class TimeVaryingPSD(object):
             self._curr_psd_index[curr_pid] = index
         return self._curr_psd[curr_pid]
 
+def parse_injection_range(num_inj, rangestr):
+    part = int(rangestr.split('/')[0])
+    pieces = int(rangestr.split('/')[1])
+    tmin =  int(num_inj / float(pieces) * part)
+    tmax =  int(num_inj / float(pieces) * (part+1))
+    return tmin, tmax
```

Comment:
Will this function guarantee that the final set of injections is identical to the starting set, i.e. is rounding/truncating going to be an issue in some edge cases? Would it be safer to specify instead directly the range of injection indices tmin/tmax?

Reply:
tbh this is copy/pasted from how this is done in the parallelization of coinc jobs - I'm fairly sure there won't be an issue with this, as tmax from part n will be the same as tmin from part n+1, provided pieces is the same, so the indexing should be fine

Reply:
specifying tmin and tmax directly wouldn't work in the case that you don't know a priori the number of injections

Reply:
Ah, I think I was unnecessarily worried by the floats. Looks like you could achieve the same just with integers though,
```python
    tmin =  num_inj * part // pieces
    tmax =  num_inj * (part+1) // pieces
```
which maybe is more clearly safe.

Reply:
updated as suggested